app/controllers/bookings_controller.rb

```diff
@@ -0,0 +1,75 @@
+class BookingsController < ApplicationController
+  before_action :set_booking, only: [:show, :edit, :update, :destroy]
+
+  # GET /bookings
+  # GET /bookings.json
+  def index
+    @bookings = Booking.all
+  end
+
+  # GET /bookings/1
+  # GET /bookings/1.json
+  def show
+  end
+
+  # GET /bookings/new
+  def new
+    @bookings = Booking.all
+  end
+
+  # GET /bookings/1/edit
+  def edit
+  end
+
+  # POST /bookings
+  # POST /bookings.json
+  def create
+    facility = Facility.first
```

Comment:
When this booking is being made, you are on a page. That page should be intimately tied to a facility. *That* is the facility that should be of concern to this booking.